engine/src/main/java/org/camunda/bpm/engine/impl/cfg/ProcessEngineConfigurationImpl.java

```diff
@@ -5058,6 +5060,14 @@ public ProcessEngineConfigurationImpl setLoggingContextTenantId(String loggingCo
     return this;
   }
 
+  public Level getBpmnStackTraceLogLevel() {
+    return bpmnStackTraceLogLevel;
+  }
+
+  public void setBpmnStackTraceLogLevel(final Level bpmnStackTraceLogLevel) {
+    this.bpmnStackTraceLogLevel = bpmnStackTraceLogLevel;
```

Comment:
We should make the setter chainable: 
```suggestion
  public ProcessEngineConfigurationImpl setBpmnStackTraceLogLevel(final Level bpmnStackTraceLogLevel) {
    this.bpmnStackTraceLogLevel = bpmnStackTraceLogLevel;
    return this;
```

Reply:
Nice catch!